### PR TITLE
[15.0][MIG] sale_order_invoice_amount

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration
@ForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

File: sale_order_invoice_amount/models/sale_order.py [hunk 1]
```diff
@@ -41,3 +40,33 @@ def _compute_invoice_amount(self):
...
+        "order_line.price_unit",
+        "amount_total",
+        "amount_untaxed",
+        "state",
+        "invoice_ids",
+        "invoice_ids.amount_total_signed",
+        "amount_total",
+        "invoice_ids.state",
+    )
+    def _compute_tax_totals_json(self):
```
Comment (rousseldenis) [inline sale_order_invoice_amount/models/sale_order.py:55]: @JasminSForgeFlow  Does this code deserves here ?

**Review by rousseldenis — APPROVED** [review]

**Comment by JasminSForgeFlow** [comment]
Hi @dreispt,

Can you please try again to merge?

Thanks

**Pull request merged**